Tests/GRPCInterceptorsTests/TracingInterceptorTests.swift

```diff
@@ -66,30 +69,123 @@ final class TracingInterceptorTests: XCTestCase {
         )
       }
 
-      var streamIterator = stream.makeAsyncIterator()
-      var element = await streamIterator.next()
-      XCTAssertEqual(element, "request1")
-      element = await streamIterator.next()
-      XCTAssertEqual(element, "request2")
-      element = await streamIterator.next()
-      XCTAssertNil(element)
-
-      var messages = response.messages.makeAsyncIterator()
-      var message = try await messages.next()
-      XCTAssertEqual(message, ["response"])
-      message = try await messages.next()
-      XCTAssertNil(message)
-
-      let tracer = InstrumentationSystem.tracer as! TestTracer
-      XCTAssertEqual(
-        tracer.getEventsForTestSpan(ofOperationName: methodDescriptor.fullyQualifiedMethod).map {
-          $0.name
-        },
-        [
-          "Request started",
-          "Received response end",
-        ]
+      await AssertStreamContentsEqual(["request1", "request2"], stream)
+      try await AssertStreamContentsEqual([["response"]], response.messages)
+
+      AssertTestSpanComponents(forMethod: methodDescriptor) { events in
+        XCTAssertEqual(
+          events.map({ $0.name }),
+          [
+            "Request started",
+            "Request ended",
+            "Received response start",
+            "Received response end",
+          ]
+        )
+      } assertAttributes: { attributes in
+        XCTAssertEqual(
+          attributes,
+          [
+            "rpc.system": .string("grpc"),
+            "rpc.method": .string(methodDescriptor.method),
+            "rpc.service": .string(methodDescriptor.service.fullyQualifiedService),
+            "rpc.grpc.status_code": .int(0),
```

Comment:
Is this necessary for the attribute values? Feels like they should be `ExpressibleBy{String,Integer}Literal`. If they aren't could you file an issue for them to be?